src/main/java/com/liferay/faces/util/product/internal/ProductBaseImpl.java

```diff
@@ -15,186 +15,58 @@
  */
 package com.liferay.faces.util.product.internal;
 
-import com.liferay.faces.util.logging.Logger;
-import com.liferay.faces.util.logging.LoggerFactory;
 import com.liferay.faces.util.product.Product;
 
 
 /**
- * All data in this class and any subclasses must be effectively immutable. In other words, the data must only be
- * modified in the constructor (and the init methods must only be called from a constructor).
- *
  * @author  Neil Griffin
  */
 public class ProductBaseImpl implements Product {
```

Comment:
@stiemannkj1 Please rename ProductBaseImpl to ProductBase and make it `abstract` to show that it is not intended to be an actual implementation.